spec/bucketing_spec.rb

```diff
@@ -108,47 +108,82 @@ def get_bucketing_id(user_id, entity_id=nil)
     expect(bucketer).to receive(:generate_bucket_value).and_return(50_000)
 
     experiment = config.get_experiment_from_key('group2_exp1')
-    expect(bucketer.bucket(experiment, 'test_user')).to be_nil
+    expect(bucketer.bucket(experiment,'bucket_id_ignored', 'test_user')).to be_nil
     expect(spy_logger).to have_received(:log).twice
     expect(spy_logger).to have_received(:log)
-      .with(Logger::DEBUG, "Assigned bucket 50000 to user 'test_user'.")
+      .with(Logger::DEBUG, "Assigned bucket 50000 to user 'test_user' with bucketing ID: 'bucket_id_ignored'.")
     expect(spy_logger).to have_received(:log)
       .with(Logger::INFO, "User 'test_user' is in no variation.")
   end
 
   it 'should call generate_bucket_value with the proper arguments during variation bucketing' do
-    expected_bucketing_id = get_bucketing_id('test_user', '111127')
+    expected_bucketing_id = get_bucketing_key('bucket_id_string', '111127')
     expect(bucketer).to receive(:generate_bucket_value).once.with(expected_bucketing_id).and_call_original
 
     experiment = config.get_experiment_from_key('test_experiment')
-    bucketer.bucket(experiment, 'test_user')
+    bucketer.bucket(experiment,'bucket_id_string', 'test_user')
   end
 
   it 'should call generate_bucket_value with the proper arguments during grouped experiment bucketing' do
-    expected_bucketing_id = get_bucketing_id('test_user', '101')
+    expected_bucketing_id = get_bucketing_key('ppid8','101')
```

Comment:
what does it mean?

Reply:
Just a normal string. Modified to show that it's different from user id